code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -31,22 +32,24 @@
 		S.take_overall_damage(anti_armour_damage*0.75, anti_armour_damage*0.25)
 	return BULLET_ACT_HIT
 
+/// PM9 standard rocket
 /obj/projectile/bullet/a84mm_he
 	name ="\improper HE missile"
 	desc = "Boom."
 	icon_state = "missile"
-	damage = 30
+	damage = 50
 	ricochets_max = 0 //it's a MISSILE
 	embedding = null
 
 /obj/projectile/bullet/a84mm_he/on_hit(atom/target, blocked=0)
 	..()
 	if(!isliving(target)) //if the target isn't alive, so is a wall or something
-		explosion(target, 0, 1, 2, 4)
+		explosion(target, 0, 1, 2, 4, flame_range = 3)
 	else
-		explosion(target, 0, 0, 2, 4)
+		explosion(target, 0, 1, 2, 4, flame_range = 3)
```

Comment:
is the flame range added onto the light range, or will the flames of this explosion not extend beyond its light explosion range?